### PR TITLE
/hidden2.html created

### PR DESCRIPTION
**Pull request by MoisesFGNeto**
Page /hidden2.html has been created. When the user clicks on the image from /hidden.html, they will be redirected to this new page (hidden2.html). This new page is also responsive. Without tools like Figma or Photoshop, it's challenging to choose the right colors, and it may not be 100% pixel-perfect. Also, some icons in the header and on the cards in skills section I couldn't find, so I replaced them with different icons. However, I believe I have cloned the prototype effectively.

Screenshot 2024-02-08 at 01 35 13

Screenshot 2024-02-08 at 01 23 28

Screenshot 2024-02-08 at 01 26 52

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
-    image.classList.remove('hidden-page');
-    image.style.cursor = 'zoom-out';
-  } else {
-    image.classList.add('hidden-page');
-    image.classList.remove('zoomed');
-    image.style.cursor = 'zoom-in';
-  }
-}
\ No newline at end of file
+// function toggleZoom() {
```
Comment (flaviostutz) [inline app.js]: Pls remove commented code if it's not used anymore

**Review by flaviostutz — CHANGES_REQUESTED** [review]
See comments

**Comment by flaviostutz** [comment]
You did it right exchanging icons by others that you found :)

File: app.js (hunk 1 above)
Comment (MoisesFGNeto) [reply app.js]: Commented code removed ;)

**Review by flaviostutz — APPROVED** [review]
Niceee!

**Pull request merged**